src/_pytest/_argcomplete.py

```diff
@@ -53,9 +53,12 @@
   which should throw a KeyError: 'COMPLINE' (which is properly set by the
   global argcomplete script).
 """
-from __future__ import absolute_import, division, print_function
-import sys
+from __future__ import absolute_import
```

Comment:
@asottile can you change the tool so for __future__ imports it does one line?

Reply:
nope, not configurable intentionally

Reply:
@asottile not asking for a configuration there ^^